modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java

```diff
@@ -536,6 +550,13 @@ private String _formatBody(
 				body),
 			!PropsValues.NOTIFICATION_EMAIL_TEMPLATE_ENABLED);
 
+		for (TemplateContextContributor templateContextContributor :
+				_serviceTrackerList) {
+
+			templateContextContributor.prepare(
+				template, ObjectActionThreadLocal.getHttpServletRequest());
```

Comment:
@rijema shouldn't you be passing `null` instead of `ObjectActionThreadLocal.getHttpServletRequest()`?

Reply:
Yes, you're right!